server/tests/userTests.js

```diff
@@ -46,5 +53,28 @@ describe('Test All user Endpoints', () => {
           done();
         });
     });
+    it('It should Sign the user in ', (done) => {
+      chai
+        .request(app)
+        .post('/api/v1/auth/signin')
+        .send(correctLoginUser)
+        .end((err, res) => {
+          res.should.have.status(200);
+          res.should.have.property('status');
```

Comment:
@mic7x4 can you expect/test the keys of the `response.body` and their values as well?